nomad/grapl-core.nomad

```diff
@@ -66,6 +66,7 @@ variable "analyzer_executor_queue" {
 variable "dgraph_replicas" {
   type    = number
   default = 1
+  # This value must be odd. Otherwise dgraph_zero will exit
```

Comment:
not a must do but just saying its possible
```
validation {
    condition     = var.dgraph_replicas % 2 == 1
    error_message = "This value must be odd. Otherwise dgraph_zero will exit"
}
```

Reply:
oooooh

Reply:
I knew that terraform supported this but I couldn't find any mention of it in the nomad docs. I've now tested it and it does work. 
I may try and put up a PR to the nomad docs for this